### ISSUE TITLE
Frontal and surrounding Light

### ISSUE THREAD

**Issue by Husky110** [lllyasviel/IC-Light#50]
That tool is really cool! :)
But we have lights from almost all angles, but not frontal or surrounding, like "I want the faces lighted up"...

**Comment by V1sionVerse** [comment]
I have the same problem. I wonder if this is an inherent limitation of this method or if my prompting is just bad. It seems the more you add "brightly lit", "sunlight", etc. the more it wants to put the sun in the background leaving the subject very underexposed.

**Comment by Husky110** [comment]
Found a post on reddit where a guy was able to control a lightsource within comfyui by positioning his mobile phone... Couldn't find it again tho :( but that seems to proof the technically possibility of a complete light-source-control.

**Comment by V1sionVerse** [comment]
Interesting, please do link it here if you can find it again!

**Comment by Husky110** [comment]
@V1sionVerse and @lllyasviel - Found it! https://www.reddit.com/r/comfyui/comments/1ctfiuo/controlling_iclight_using_my_phones_gyroscope_via/

**Comment by V1sionVerse** [comment]
Thanks, I guess ComfyUI it is for now 😅

**Comment by Husky110** [comment]
Not necessarily... This project is actually a pretty sophisticated background-changer... :sweat_smile:

**Comment by loboere** [comment]
It worked a little for me to train a lora with faces that have a front light, then I use the trigger word and it seems to work

**Comment by loboere** [comment]
It would be great if you could use reference images.

**Comment by diodiogod** [comment]
I'm pretty sure you can use img2img with the light source on forge, just like on confyui. Add the light (white mask) to area of the face the way you want.
